read-fonts/src/tables/fvar.rs

```diff
@@ -17,6 +18,55 @@ impl<'a> Fvar<'a> {
     pub fn instances(&self) -> Result<ComputedArray<'a, InstanceRecord<'a>>, ReadError> {
         Ok(self.axis_instance_arrays()?.instances())
     }
+
+    /// Converts user space coordinates provided by an unordered iterator
+    /// of `(tag, value)` pairs to normalized coordinates in axis list order.
+    ///
+    /// Stores the resulting normalized coordinates in the given slice.
+    ///
+    /// * User coordinate tags that don't match an axis are ignored.
+    /// * User coordinate values are clamped to the range of their associated
+    ///   axis before normalization.
+    /// * If more than one user coordinate is provided for the same tag, the
+    ///   last one is used.
+    /// * If no user coordinate for an axis is provided, the associated
+    ///   coordinate is set to the normalized value 0.0, representing the
+    ///   default location in variation space.
+    ///
+    /// If the `avar` table is provided, applies remapping of coordinates.
+    pub fn user_to_normalized(
```

Comment:
we should document the expectations of the `normalized_coords` argument. I assume we expect it to be of a given length, and that we will overwrite the entire contents?